### ISSUE TITLE
Add option to delete docs embed

### ISSUE THREAD

**Issue by gustavwilliam** [python-discord/bot#1098]
When `!docs print` is run, python returns the documentation for the print function.

Often times, the docs returned may not be what was expected or the user simply wanted to do a quick check and then get rid of the large embed. In both of those cases, the option to delete the response would be perfect.

This is already implemented for commands such as `!eval`, where a 🗑 reaction can be clicked to delete the response. The logic could simply be carried over.

**Comment by AtieP** [comment]
I can do this

**Comment by MarkKoz** [comment]
Yeah, this is a good idea. We already do this for a lot of other command outputs, so I don't see a reason to leave docs out. Let's do this.

**Comment by AtieP** [comment]
This issue has been resolved.
